### PR TITLE
Add OceanPop - A HunniePop Clone

### PR DESCRIPTION
**Pull request by naddoska**
Hello Osgameclones Team! 

I added @sharkwouter s HunniePop Clone called 'OceanPop' to your List of Clones! 
Hope I did everything right. 
Have a good day :)

### CODE REVIEW

File: games/o.yaml [hunk 1]
```diff
@@ -13,6 +13,22 @@
...
+  licenses:
+  - MIT
+  development: sporadic
+  originals:
+  - HuniePop
+  info: HuniePop without the Hunnies
+  repo: https://github.com/sharkwouter/oceanpop
+  status: playable
+  type: clone
+  updated: 2022-03-09
```
Comment (cxong) [inline games/o.yaml:30]:
```suggestion
  updated: 2022-10-23
```

File: games/o.yaml [hunk 2]
```diff
@@ -13,6 +13,22 @@
...
+- name: OceanPop
+  images:
+  - https://raw.githubusercontent.com/sharkwouter/oceanpop/1.0/screenshot.jpg
+  langs:
+  - C++
+  licenses:
+  - MIT
+  development: sporadic
+  originals:
+  - HuniePop
```
Comment (cxong) [inline games/o.yaml:25]: I think something like bejeweled would be better here; I don't see any similarity with huniepop specifically except for the name "pop"
wondering what @sharkwouter thinks

File: games/o.yaml [hunk 3]
```diff
@@ -13,6 +13,22 @@
...
   updated: 2016-02-08
 
+- name: OceanPop
+  images:
+  - https://raw.githubusercontent.com/sharkwouter/oceanpop/1.0/screenshot.jpg
+  langs:
+  - C++
+  licenses:
+  - MIT
+  development: sporadic
```
Comment (cxong) [inline games/o.yaml:23]:
```suggestion
  development: active
```

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/o.yaml (hunk 2 above)
Comment (sharkwouter) [reply games/o.yaml:25]: Sure, I can eloborate. In Huniepop you have a specific amount of moves in which you have to reach a target score and you are able to move gems as far as you'd like. OceanPop implements both of those mechanics. This makes it quite different from Bejeweled.

**Comment by naddoska** [comment]
@cxong I updated the requested changes :)

**Review by cxong — APPROVED** [review]

**Pull request merged**